airbyte-integrations/connectors/source-salesloft/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 41991d12-d4b5-439e-afd0-260a31d4c53f
-  dockerImageTag: 1.1.1
+  dockerImageTag: 1.1.2
```

Comment:
Same comment as above: https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors